test/src/engine/BMGameTest.php

```diff
@@ -5759,6 +5773,10 @@ public function test_berserk_round() {
         $this->assertGreaterThanOrEqual(1, $game->actionLog[1]->params['postAttackDice']['attacker'][0]['value']);
 
         $game->update_game_state();
+        $this->assertEquals(BMGameState::CHOOSE_TURBO_SWING, $game->gameState);
```

Comment:
Is it expected that games without turbo dice should move through `BMGameState::CHOOSE_TURBO_SWING`?  I didn't think games without chance/focus dice moved through `BMGameState::REACT_TO_INITIATIVE`, but maybe i'm wrong about that.